Components/Card/card.html

```diff
@@ -0,0 +1,262 @@
+                class="fas fa-times-circle"></i></span></h1>
+          <ul class="list-subtopic">
+            <li>
+              <a class="link-hover" href="../../Documentation/Introduction.html">Introduction</a>
+            </li>
+            <li><a class="link-hover" href="../../Documentation/Color.html">Colors</a></li>
+            <li>
+              <a class="link-hover" href="../../Documentation/Typography.html">Typography</a>
+            </li>
+            <li>
+              <a class="link-hover" href="../../Documentation/Conclusion.html">Conclusion</a>
+            </li>
+          </ul>
+        </div>
+        <div>
+          <h1 class="list-topic">Components</h1>
+          <ul class="list-subtopic">
+            <li>
+              <a class="link-hover" href="../Avatar/Avatar.html">Avatar</a>
+            </li>
+            <li>
+              <a class="link-hover" href="../Badge/Badge.html">Badge</a>
+            </li>
+            <li>
+              <a class="link-hover" href="../Alert/Alert.html">Alert</a>
+            </li>
+            <li>
+              <a class="link-hover" href="../Button/Button.html">Button</a>
+            </li>
+            <li><a class="link-active" href="./card.html">Card</a></li>
+            <li><a class="link-hover" href="../Image/image.html">Image</a></li>
+            <li><a class="link-hover" href="../Input/input.html">Input</a></li>
+            <li><a class="link-hover" href="../text-utilites/text-utilites.html">Text Utilities</a></li>
+            <li><a class="link-hover" href="../lists/lists.html">Lists</a></li>
+            <li><a class="link-hover" href="../navigation/navigation.html">Navigation</a></li>
+            <li><a class="link-hover" href="../modal/modal.html">Modal</a></li>
+            <li><a class="link-hover" href="../rating/rating.html">Rating</a></li>
+            <li><a class="link-hover" href="../toast/toast.html">Toast</a></li>
+            <li><a class="link-hover" href="../grid/grid.html">Grid System</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="content">
+      <i id="sidebar-toggle" class="fas fa-bars hamburger"></i>
+      <h1 class="content-title">Card</h1>
+      <p class="sub-content">
+        Cards are surfaces that display content and actions on a single topic.
+        They should be easy to scan for relevant and actionable information. Elements, like text and images, should be
+        placed on them in a way that clearly indicates hierarchy.
+      </p>
+      <div class="divider"></div>
+      <h1 class="content-title">Card with Badge</h1>
+      <p class="sub-content">
+        A badge on card is used to show something important about a product in single word or a icon.
+        <div class="content-container">
+          <div class="card-container">
+            <img src="../../Assets/card-image-1.jpg" alt="A graphic card image">
```

Comment:
Nice work. Just one suggestion, I think, you can use `loading=lazy` with the `<img />` to improve the performance.
Everything else looks good.

Reply:
Thanks for the suggestion. i will definitely add it!